### ISSUE TITLE
2019-12-10 M1SS computation error when all concentrations set to NA

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#98]
### Error

When using tc008, tc011, tc012 M1SS and all concentrations set to NA (i.e. `d$PKCNCN <- rep(NA, nrow(d))`) the subsequent computation error occurs.

```
 Error in value[[3L]](cond) : Error: object 'cest_kel' not found
For SDEID 24280050 
7. stop(paste0(e, "For SDEID ", unique(data_data[, map_data$SDEID])[i])) at run_M1_SS_computation.R#2740
6. value[[3L]](cond) 
5. tryCatchOne(expr, names, parentenv, handlers[[1L]]) 
4. tryCatchList(expr, classes, parentenv, handlers) 
3. tryCatch({
    if (comp_required[["DOSECi"]] || comp_required[["DOSEC"]]) {
        dose_ci <- list()
    } ... at run_M1_SS_computation.R#1040
2. run_M1_SS_computation(data = merged_data, map = map_data, method = method, 
    model_regex = model_regex, parameter_list = parameter_list, 
    return_list = return_list) at run_computation.R#2354
1. run_computation(data = d, map = mct, flag = flags, parameterset = parameterset) 
```

The parametersets for each of these test cases are included below

#### tc008 M1SS

```
"AUCALL;AUCALLDN;AUCINFO;AUCINFOC;AUCINFODN;AUCINFP;AUCINFPC;AUCINFPDN;AUCLAST;AUCLASTi;AUCLASTDNi;AUCLASTCi;AUCT;AUCT1_T2;AUCTAUi;AUCTAUDNi;AUCXPCTO;AUCXPCTP;AUMCTAUi;CAVi;CLASTi;CLFO;CLFTAUi;CLFTAUWi;CLO;CLRTAUi;CMAXi;CMAXCi;CMAXDNi;CMINi;DOSECi;CTROUGHi;CTROUGHENDi;DIi;DOSEi;KEL;KELNOPT;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;MRTEVIFOi;MRTEVIFPi;PTFi;PTRi;PTROUGHRi;PTROUGHREND;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TMAXi;TMINi;TOLD;VZFP;VZFTAUi;VZFTAUWi;FLGACCEPTKEL;FLGACCEPTKELCRIT;FLGACCEPTPREDOSE;FLGACCEPTPREDOSECRIT;FLGACCEPTTMAX;SPANRATIOCRIT;FLGACCEPTKEL;FLGACCEPTTAU"
```

#### tc011 M1SS

```
"AUCALL;AUCDN;AUCINFO;AUCINFOC;AUCINFODN;AUCINFP;AUCINFPC;AUCINFPDN;AUCLASTi;AUCLASTDN;AUCLASTDNi;AUCLASTC;AUCLASTCi;AUCT;AUCT1_T2;AUCTAUi;AUCTAUDNi;AUCXPCTO;AUCXPCTP;AUMCTAUi;CAVi;CLASTi;CLFO;CLFTAUi;CLFTAUWi;CMAXi;CMAXDNi;CMINi;DOSEC;CTROUGHi;CTROUGHENDi;DIi;DOSEi;KEL;KELC0;KELNOPT;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;MRTLAST;MRTLASTi;MRTEVIFOi;MRTEVIFPi;PTFi;PTR;PTRi;PTROUGHR;PTROUGHRi;PTROUGHREND;PTROUGHRENDi;TAUi;THALF;THALFF;TLAG;TLAST;TMAXi;TMINi;TOLDi;VZFP;VZFTAUi;VZFTAUWi;FLGACCEPTKEL;FLGACCEPTTAU"
```

#### tc012 M1SS

```
"AUCALL;AUCALLDN;AUCINFO;AUCINFOC;AUCINFODN;AUCINFP;AUCINFPC;AUCINFPDN;AUCLAST;AUCLASTi;AUCLASTDNi;AUCLASTCi;AUCT;AUCT1_T2;AUCTAUi;AUCTAUDNi;AUCXPCTO;AUCXPCTP;AUMCTAUi;CAVi;CLASTi;CLFO;CLFTAUi;CLFTAUWi;CLO;CLRTAUi;CMAXi;CMAXCi;CMAXDNi;CMINi;DOSECi;CTROUGHi;CTROUGHENDi;DIi;DOSEi;KEL;KELNOPT;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;MRTEVIFOi;MRTEVIFPi;PTFi;PTRi;PTROUGHRi;PTROUGHREND;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TMAXi;TMINi;TOLD;VZFP;VZFTAUi;VZFTAUWi;FLGACCEPTKEL;FLGACCEPTTAU"
```

**Comment by jhhughes256** [comment]
As of 00cdcd3 this issue still occurs.

**Comment by rudraya** [comment]
This issue was addressed in 560bcad, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
This no longer occurs as of 560bcad.
